testing/scheme/scheme.go

```diff
@@ -0,0 +1,19 @@
+package scheme
+
+import (
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	cisv1 "github.com/rancher/cis-operator/pkg/apis/cis.cattle.io/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubectl/pkg/scheme"
+)
+
+func Scheme() *runtime.Scheme {
+	baseScheme := scheme.Scheme
+	return AddCRDsToScheme(baseScheme)
+}
+
+func AddCRDsToScheme(s *runtime.Scheme) *runtime.Scheme {
+	cisv1.AddToScheme(s)
```

Comment:
To add a CRD, just add it to the scheme here. Most packages for controllers have AddToScheme defined (definitely true for `rancher/wrangler` based controllers).